app/static/messages.properties

```diff
@@ -300,9 +307,8 @@
     "cluster_form_hostgroup_label" : "Hostgroup configuration",
     "cluster_form_hostgroup_template_label" : "Template",
     "cluster_form_hostgroup_group_size_label" : "Group size",
-    "cluster_form_email_label" : "Email notification when cluster is provisioned",
-    "cluster_form_dedicated_label" : "Dedicated instances",
-    "cluster_form_ambari_repo_label" : "Ambari Repository config",
+    "cluster_form_email_label" : "Send email when cluster is ready",
```

Comment:
We also send error emails too, so this label could be 'Send email notifications'.